packages/gobba/gobba.0.4.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "sudo-woodo3@protonmail.com"
+authors: ["Alessandro Cheli"]
+homepage: "https://github.com/0x0f0f0f/gobba"
+bug-reports: "https://github.com/0x0f0f0f/gobba/issues"
+dev-repo: "git+https://github.com/0x0f0f0f/gobba.git"
+license: "MIT"
+synopsis: "A simple, didactical, purely functional programming language"
+description: "A simple, didactical, purely functional programming language written for the programming 2 course at the University of Pisa, extended with a simple parser made with Menhir and ocamllex"
+doc: "https://0x0f0f0f.github.io/gobba"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test" "-j" jobs] {with-test}
```

Comment:
```suggestion
  ["sh" "-c" "GOBBA_EXAMPLES=$(realpath ./examples/) dune runtest -p %{name}% -j %{jobs}%"] {with-test}
  ["sh" "-c" "bisect-ppx-report -html coverage/ -I _build/default _build/default/test/bisect*.out"] {with-test}
```
This should be much better.
What is in your makefile is not correct for releases:
```
dune clean
dune build -j $(JOBS)
dune build @install
GOBBA_EXAMPLES=$(realpath ./examples/) dune runtest -f
bisect-ppx-report -html coverage/ -I _build/default _build/default/test/bisect*.out;
```
First of all, dune should never be called in releases without `-p name` or `--profile=release`, this might break this package in the future as dune turns some warnings as error by default. Then the package is built twice (dune build -p name -j jobs above, then dune clean & dune build again), this is unnecessary.

Reply:
Thank you for your dozens of suggestions!